IndicoV2/Extensions/Jobs/JobAwaiter.cs

```diff
@@ -27,7 +27,8 @@ public async Task<JToken> WaitReadyAsync(string jobId, TimeSpan checkInterval, C
 
             if (status != JobStatus.SUCCESS)
             {
-                throw new JobNotSuccessfulException(status);
+                var failReason = await _jobsClient.GetFailReasonAsync(jobId);
```

Comment:
This logic overall needs some refinement, given there are multiple statuses like PENDING_AUTOREVIEW, or if perhaps someone calls it on a doc still in PROCESSING. We don't want to fail on that.

Reply:
`PENDING_AUTOREVIEW` is from the `SubmissionStatus`, and here I'm checking the `JobStatus`. 

Anyway - you were right there are statuses being missing here, I'm changing this part to include `PENDING, RECEIVED, STARTED` as those in progress.

Reply:
Ah right, got the enum list confused :grimacing: But thanks for fixing it!